fried-chicken.txt

```diff
@@ -0,0 +1,5 @@
+- Cure chicken with salt
+- Add egg, buttermilk, hot sauce, pickle juice
+- Add equal parts flour to kosher salt (cups to tsp)
+- Dunk chicken into flour, wet, flour
+- Fry
```

Comment:
Did you think about adding the temperature of the oil? What type of oil? Temperature of the chicken when it's done?